src/Component/Symbolizer/Renderer/Renderer.tsx

```diff
@@ -80,10 +82,14 @@ export class Renderer extends React.Component<RendererProps> {
   }
 
   updateFeature() {
+    const vectorData = this.props.data as VectorData;
```

Comment:
[DataUtil](https://github.com/terrestris/geostyler/blob/master/src/Util/DataUtil.ts) provides functions for checking the type of data (vector/raster). You might want to use it here.